src/Twilio/Version.php

```diff
@@ -109,7 +109,7 @@ public function fetch(string $method, string $uri,
             $timeout
         );
 
-        if ($response->getStatusCode() < 200 || $response->getStatusCode() >= 300) {
+        if ($response->getStatusCode() < 200 || $response->getStatusCode() >= 400) {
```

Comment:
Were you able to verify locally that this fixes the issue you're seeing?

Reply:
Also, could you add a comment about why 3XX is allowed _just_ here and not in the other wrappers?

Reply:
I can verify that this code change resolves the issue for me if I change the file locally.  To @childish-sambino's comment, this seems to be a unique endpoint response in that it is a redirect, I don't believe any of the other endpoints are redirects which is why 3XX is not expected there.

Reply:
Yup, just meant a comment in the code itself.

Reply:
I added a comment.  I think that's what you wanted.